src/Temporalio/Client/TestService.cs

```diff
@@ -0,0 +1,41 @@
+using System.Threading.Tasks;
+using Google.Protobuf;
+
+namespace Temporalio.Client
+{
+    /// <summary>
+    /// Raw gRPC test service.
+    /// </summary>
+    public abstract partial class TestService : RpcService
+    {
+        internal override Bridge.Interop.RpcService Service => Bridge.Interop.RpcService.Test;
+
+        /// <summary>
+        /// Implementation of the test service.
+        /// </summary>
+        public class Impl : TestService
```

Comment:
Why is this inner `Impl` class required?

Reply:
So that users could have their own impls of these services if they wanted to for mocks or whatever.